### DISCUSSION TITLE
Configure user policies

### DISCUSSION THREAD

**Discussion by sumarnaz** [minio/minio#11703] [Q&A]
Hello.
I'm trying to setup minio as a multiuser storage service.
What I would like to obtain is:
1) Add a user (bob)
2) Give bob the permission to create one or more buckets
3) Give bob the permission to list and use it's own created buckets only.

In the example mentioned in the document: https://docs.minio.io/docs/minio-multi-user-quickstart-guide.html, the policy assigns  s3:GetObject on a given bucket, but this is not what I want, since I don't know how many buckets the user will create and their names.

Can anybody give me some suggestion or point me to some example?

Thanks a lot.
Riccardo

**Comment by harshavardhana** [comment]
> * Add a user (bob)
> * Give bob the permission to create one or more buckets
> * Give bob the permission to list and use it's own created buckets only.

There is no explicit ownership on the buckets - so it's all policy-driven use the right resource for the bucket in your policy.

**Reply by sumarnaz** [reply]
Hello.
Thanks for your answer.
Can you please give me an example of a resource definition to make that policy?
Thanks

**Reply by harshavardhana** [reply]
https://docs.aws.amazon.com/AmazonS3/latest/userguide/example-policies-s3.html
